### PR TITLE
Create conda-forge package for rbc

### PR DESCRIPTION
**Pull request by pearu**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pearu** [comment]
Confirming to be a maintainer.

**Comment by pearu** [comment]
@conda-forge/help-python the [rbc](https://github.com/xnd-project/rbc/) conda package is ready for review.

File: recipes/rbc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/xnd-project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: c9566d7e820e684dbcac01baa11a7fde5048ecbfcea0d0d929ebfdea764a542a
+
+build:
+  number: 0
+  skip: True  # [py<35 or win]
+  script: "{{ PYTHON }} setup.py install"
```
Comment (isuruf) [inline recipes/rbc/meta.yaml]: See https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

File: recipes/rbc/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (isuruf) [inline recipes/rbc/LICENSE.txt]: Can you fix the sdist upstream by adding the license file location in `MANIFEST.in` ?

File: recipes/rbc/meta.yaml (hunk 1 above)
Comment (pearu) [reply recipes/rbc/meta.yaml]: Fixed, now using pip to install the package.

File: recipes/rbc/LICENSE.txt (hunk 2 above)
Comment (pearu) [reply recipes/rbc/LICENSE.txt]: Done in upstream.

File: recipes/rbc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+build:
+  number: 0
+  skip: True  # [py<35 or win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
```
Comment (isuruf) [inline recipes/rbc/meta.yaml]: Move the above 2 lines to `host`

Comment (pearu) [reply recipes/rbc/meta.yaml]: Done.

File: recipes/rbc/LICENSE.txt (hunk 2 above)
Comment (isuruf) [reply recipes/rbc/LICENSE.txt]: Can you remove this file then?

Comment (pearu) [reply recipes/rbc/LICENSE.txt]: Sure. Removed.

File: recipes/rbc/meta.yaml [hunk 4]
```diff
@@ -41,7 +41,6 @@ about:
   home: https://github.com/xnd-project/rbc
   license: BSD-3-Clause
   license_family: BSD
-  license_file: LICENSE.txt
```
Comment (isuruf) [inline recipes/rbc/meta.yaml]: You still need to keep this line and conda-build will find it from the source tarball

Comment (pearu) [reply recipes/rbc/meta.yaml]: Fixed, sorry for the confusion.

**Pull request merged**